storefront/app/views/workarea/storefront/pages/robots.text.erb

```diff
@@ -1,5 +1,6 @@
 User-Agent: *
 Allow: <%= sitemap_path %>
+Sitemap: https://<%= Workarea.config.host %>/sitemap.xml.gz
```

Comment:
We really need a `Workarea.protocol`

Reply:
that would be really nice. when i was working with site-builder on `https://mysitebuildersite.test`, my previews would look terrible because the CSS/JS assets were trying to be fetched from `http://somesite-preview.mysitebuildersite.test`. developers using puma-dev are going to hit that in the real world if they use an https URL...